### PR TITLE
Add headings to Tips & Tricks

### PR DESCRIPTION
**Pull request by gilgongo**
As per https://github.com/jamulussoftware/jamuluswebsite/issues/730

### CODE REVIEW

File: wiki/en/Tips-Tricks-More.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ permalink: "/wiki/Tips-Tricks-More"
 
 Jamulus user [Chris Rimple](https://sourceforge.net/u/chrisrimple/profile/) has compiled a massive amount of information relating to [Remote Band Rehearsals](https://docs.google.com/document/d/1smcvsxdaaViPQvGMQHmah_6BQeqowhmGSFMHfnlY2FI/) (Google doc), which covers topics such as hardware and software configuration including examples and advice for newcomers to the field. He also includes a section on Jamulus in comparison to other solutions.
 
+# Using the Jamulus Client
```
Comment (ann0see) [inline wiki/en/Tips-Tricks-More.md]:
```suggestion
## Using the Jamulus Client
```
As for HTML: We should not use multiple h1 headings.

File: wiki/en/Tips-Tricks-More.md [hunk 2]
```diff
@@ -102,6 +96,16 @@ Make sure you connect your MIDI device's output port to the Jamulus MIDI in port
 
 *Tip*: When you enable MIDI control in Jamulus, each user's name is prepended with a number, with the leftmost user starting at 0, then 1, etc. With default settings, when some users leave and others join, their left-right arrangement in the GUI may cease to follow a numerical order, making it more difficult to know who each physical fader/knob on your MIDI controller corresponds to. In order to keep the fader strips following a numerical order, go to "View" on the top menu bar and choose "Sort Users by Name".
 
+# For Server admins
```
Comment (ann0see) [inline wiki/en/Tips-Tricks-More.md]:
```suggestion
## For Server admins
```

File: wiki/en/Tips-Tricks-More.md [hunk 3]
```diff
@@ -102,6 +96,16 @@ Make sure you connect your MIDI device's output port to the Jamulus MIDI in port
...
+
+## Converting a public Server to a private one on the fly
```
Comment (ann0see) [inline wiki/en/Tips-Tricks-More.md]: Should be updated to three levels

File: wiki/en/Tips-Tricks-More.md [hunk 4]
```diff
@@ -102,6 +96,16 @@ Make sure you connect your MIDI device's output port to the Jamulus MIDI in port
...
+
+You can run a public Server long enough for your band to connect, then go private by simply unchecking the 'Make my Server Public' box in the Server GUI. Your band mates will still be connected to the Server until they disconnect. (Thanks to [David Savinkoff](https://github.com/DavidSavinkoff) for this tip!)
+
+## Remote management of recordings
+
+Jamulus user [vdellamea](https://github.com/vdellamea) has written a [web-based remote tool](https://github.com/vdellamea/jamulus-server-remote) for starting and stopping recordings on Linux Servers, allowing you to then download them from your browser. See also [Jamulus Jam Exporter](https://github.com/pljones/jamulus-jamexporter) by [pljones](https://github.com/pljones), which also includes a Server recording recovery script.
```
Comment (ann0see) [inline wiki/en/Tips-Tricks-More.md:107]: JSON-RPC could/should be added too.

Comment (gilgongo) [reply wiki/en/Tips-Tricks-More.md:107]: Not sure I'm the person to do that though.

Comment (ann0see) [reply wiki/en/Tips-Tricks-More.md:107]: Just add something like "Jamulus also supports JSON-RPC for controlling client and server. Please have a look at the API-documentation (link to JSON-RPC.md)"

Comment (gilgongo) [reply wiki/en/Tips-Tricks-More.md:107]: For it to be a "tip" it needs to actually solve a problem, or help you do something specific though. Simply linking to some docs isn't what this page is about.

Comment (ann0see) [reply wiki/en/Tips-Tricks-More.md:107]: Hmm. Probably? It's still somewhat related though.

Comment (gilgongo) [reply wiki/en/Tips-Tricks-More.md:107]: So for example, you can use MIDI with `ctrlmidich` to control fader, pan control and mute and solo buttons. If there's something specific like that we can write a HOWTO for using JSON-RPC, then maybe put that in?

**Pull request merged**